### ISSUE TITLE
No longer functional with new version of the Default weather module

### ISSUE THREAD

**Issue by SlamminS85** [MatthiPi/MMM-DefaultWeatherAnimated#10]
This module is no longer functional with the latest version of the default weather module. Weather information will always display as perpetually loading.

The weather information is actually there and is being sent to other modules that use weather information, but the weather information is never displayed.

**Comment by rmcon** [comment]
Ah, so it's not just me.  I'm not the most tech savvy person and just trying my hand at MM, I was thinking I had gooned up something when it wouldn't work for me.  Guess I'll reset the old files and check periodically for updates.

**Comment by SlamminS85** [comment]
No, not just you. I compared previous version of the default weather module and this module and was unable to determine what change caused this issue. I suspect a race condition existed in the previous version, but until the new Default weather module was updated, it never resulted in perpetual loading. 

I have since moved on to other things and the default icons are just fine.
